src/ui.cpp

```diff
@@ -818,16 +820,20 @@ bool uilist::scrollby( const int scrollby )
             }
         }
     } else {
-        if( fselected >= static_cast<int>( fentries.size() ) ) {
+        if( fselected >= recmax ) {
             fselected = 0;
         }
+        int x = 0;
```

Comment:
@Pupsi-Mupsi What's this part supposed to do?

I don't understand the comment in german or why this change is only on forwards branch but not the `backwards` one, but this change breaks rolling "forward" over disabled menu items with down arrow, easiest repro would be activating read menu on a book that's lower than player's existing skill level - you'll then be able to select disabled menu item.

If there's more menu items after a disabled one you won't be reaching them via down/pgdown keys, only up ones

Rolling back just this part makes selection work normally including the pgup/down functionality you added

Reply:
;-) Ups. My bad. The comment wasn't supposed to stay in there. I'll remove it as soon as I manage to fix the issue at hand.
As far as I remember I needed a way to workaround disabled entries in the "execute which action" menu.

Suggestions are welcome.

Thanks for making me aware of that bug.

Reply:
The thing I tested was just simple rollback of those lines except `recmax` substitute and it seemed to work as expected - menus with disabled entries were browseable via arrows and pageup/down, but may be I'm missing the scenario you tested

Reply:
This is where I had some trouble. The last two entries were browsable - so implemented this rather poor fix.

![grafik](https://user-images.githubusercontent.com/44737997/97812105-55bff580-1c7f-11eb-9331-a0d10c9f5f2f.png)